See Mozilla bug #451007 - Reload page causing problems with
RealPlayer/Helix plugin.
We can potentially drop this patch since RealPlayer is no longer being
supported, but some people may still be using it.

diff --git a/dom/base/nsObjectLoadingContent.cpp b/dom/base/nsObjectLoadingContent.cpp
--- a/dom/base/nsObjectLoadingContent.cpp
+++ b/dom/base/nsObjectLoadingContent.cpp
@@ -2902,7 +2902,8 @@
               bool aDelayedStop)
 {
   // Don't delay stopping QuickTime (bug 425157), Flip4Mac (bug 426524),
-  // XStandard (bug 430219), CMISS Zinc (bug 429604).
+  // XStandard (bug 430219), CMISS Zinc (bug 429604),
+  // RealPlayer/Helix (bug 451007).
   if (aDelayedStop
 #if !(defined XP_WIN || defined MOZ_X11)
       && !aInstanceOwner->MatchPluginName("QuickTime")
@@ -2910,6 +2911,8 @@
       && !aInstanceOwner->MatchPluginName("XStandard plugin")
       && !aInstanceOwner->MatchPluginName("CMISS Zinc Plugin")
 #endif
+      && !aInstanceOwner->MatchPluginName("RealPlayer")
+      && !aInstanceOwner->MatchPluginName("Helix")
       ) {
     nsCOMPtr<nsIRunnable> evt =
       new nsStopPluginRunnable(aInstanceOwner, aContent);
